src/main/scala/org/antipathy/scoozie/GeneratedArtefacts.scala

```diff
@@ -25,13 +23,12 @@ final class GeneratedArtefacts(workflow: Workflow,
   def validate(): Unit = {
     Scoozie.Test.validate(workflow)
     coordinatorOption.foreach(Scoozie.Test.validate)
-    if (validationStringOption.isDefined) {
-      val validationString = validationStringOption.get
-      val traversalString = Scoozie.Test.workflowTesterWorkflowTestRunner(workflow).traversalPath
 
-      if (!validationString.equals(traversalString)) {
+    validationStringOption.foreach { vString =>
+      val traversalString = Scoozie.Test.workflowTesterWorkflowTestRunner(workflow).traversalPath
+      if (!vString.equals(traversalString)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of untyped comparison operator](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)